### PR TITLE
fix LS_CSR prefix sum by using LargeArray

### PR DESCRIPTION
**Pull request by meyerzinn**
`PrefixSum` needs a cache line-aligned temporary buffer.

### CODE REVIEW

**Review by tewaro — APPROVED** [review]

**Review by tewaro — CHANGES_REQUESTED** [review]
Revert the disabling of the debug build

File: libgalois/test/graph-compile-lscsr.cpp [hunk 1]
```diff
@@ -116,5 +116,16 @@ int main() {
   // ...
   GALOIS_ASSERT(g[8] == 7);
 
+  {
+    uint64_t num_vertices = (1 << 22) + 67;
+    galois::graphs::LS_LC_CSR_Graph<void, void> big(num_vertices);
+    for (uint64_t i = 0; i < num_vertices; ++i) {
+      big.addEdgesTopologyOnly(i, {(1 + 1) % num_vertices});
```
Comment (patrickkenney9801) [inline libgalois/test/graph-compile-lscsr.cpp:123]: is this supposed to be `(1 + 1)`?

**Comment by patrickkenney9801** [comment]
please add back that deleted file either to regular git if you want to remain sane, or buggy lfs if you want the pr to drag on for 3 months

**Comment by tewaro** [comment]
I thought I did add the file.

**Comment by patrickkenney9801** [comment]
also it looks like it is a local issue for you as far as I can tell
maybe try turning it off and on? `rm -rf SRC_DIR`, the image should be fine
just remember to create `mkdir SRC_DIR/.ccache`

**Comment by tewaro** [comment]
I renamed it to a better name.

File: libgalois/test/graph-compile-lscsr.cpp (hunk 1 above)
Comment (meyerzinn) [reply libgalois/test/graph-compile-lscsr.cpp:123]: lol yes but it does not matter since the prefix sum just computes based on the number of edges.

**Comment by meyerzinn** [comment]
The issue is actually with `PrefixSum`.

**Pull request closed without merging**